data/services/install/create-pinned-image-sets.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Create PinnedImageSets service
+Wants=network.target
+After=network-online.target
+Before=stop-local-registry.service
+ConditionPathExists=!/etc/crio/crio.conf.d/50-pinned-images
```

Comment:
So is it indeed good enough when installing a multi-node cluster? I mean, do we fail gracefully on the other nodes?
If not, we can add some indication to ensure that the script runs only on a single node (e.g. as done with [node-zero](https://github.com/openshift/installer/blob/3868a1458009ddeb8633fddb535c758f8f729f4e/data/data/agent/systemd/units/assisted-service-pod.service#L5)).